### ISSUE TITLE
i18n: Mark up Prometheus List View and Raw Prometheus for translations

### ISSUE THREAD

**Issue by ifrost** [grafana/grafana#74307]
### Requirements ###

Mark up https://github.com/grafana/grafana/blob/v10.1.0/public/app/features/explore/PrometheusListView and https://github.com/grafana/grafana/tree/v10.1.0/public/app/features/explore/RawPrometheus components for translations

### Background

Please follow #70957 for more info and guidelines.

**Comment by ashutosh887** [comment]
@ifrost Should I try this?
Please assign

**Comment by ifrost** [comment]
> @ifrost Should I try this? Please assign

Thanks @ashutosh887 🙌  This component is owned by @grafana/observability-metrics squad. 

@grafana/observability-metrics do you have the capacity to provide help/review on this one?

@ashutosh887 Thank you for your interest in other issues! Please make sure you pick one at the time.

**Comment by ifrost** [comment]
After an internal discussion, we concluded that unfortunately, we need to postpone translating Explore's core visualizations as they (or some of their parts) may require externalizing to separate plugins. I18n is available only to core functionalities. There's no i18n support for external plugins, so translating Explore's visualizations now would create technical debt by creating dependencies with core, slow down further improvements, and very likely some of the work would need to be rolled back eventually.

We've decided that we need to close issues translating Explore's core visualizations and close all related pull requests. Thank you for all the time and effort you put into creating this PR. I'm sorry we need to close it, but I want to say we appreciate your work - it helped us identify this problem sooner.
